docs/platforms/javascript/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
```

Comment:
sidebar order between python and javascript is very different:
<img width="294" alt="SCR-20241202-ieql" src="https://github.com/user-attachments/assets/529285da-b82e-42bf-b1f3-12a48f1c7e04">
<img width="298" alt="SCR-20241202-iepw" src="https://github.com/user-attachments/assets/34d8b0b9-02ba-4bed-a357-117f9ebe9669">